src/app/campaign/store/models/flight-overlap.models.ts

```diff
@@ -0,0 +1,22 @@
+import { Flight } from './flight.models';
+
+export interface FlightOverlap {
+  flightId: number;
+  filters: string;
+  overlapping: Flight[];
+  loading?: boolean;
+  error?: any;
+}
+
+export const overlapFilters = (flight: Flight): string => {
+  if (flight && flight.set_inventory_uri && flight.zones && flight.zones.length && flight.endAt && flight.startAt) {
+    return [
+      `inventory=${flight.set_inventory_uri.split('/').pop()}`,
+      `zones=${flight.zones.map(z => z.id).join(',')}`,
+      `before=${flight.endAt.toISOString()}`,
```

Comment:
Seeing that the `before` filter (apps/model/flight.rb) is using the inclusive comparator `<=` in the `Flight#before` scope.  I _think_ we decided that we're using inclusive an end-at in the UI (e.g. calendar widget), which is the `endAtFudged`. If that's right than this should be `flight.endAtFudged`.

Reply:
Good catch!  Will update that.

Reply:
Okay, after digging in a bit more ...

So I select `10/19` and `10/30` in the UI.  As-is, this branch fires off a request with filters:

```
before=2020-10-31T00:00:00.000Z
after=2020-10-19T00:00:00.000Z
```

Which get run through the Flight.rb scopes: 

```ruby
scope :before, ->(time) {
  where('flights.start_at <= ?', time)
}
scope :after, ->(time) {
  where('flights.start_at >= ? OR flights.end_at >= ?', time, time)
}
```

And we end up with SQL:

```sql
WHERE start_at <= '2020-10-31T00:00:00Z'
AND (start_at >= '2020-10-19T00:00:00Z' OR end_at >= '2020-10-19T00:00:00Z')
```

So any returned flights have to start _sometime_ before midnight on the 31st.  Which I believe is already what we want?  Return all flights with a start_at anytime on the 30th.

If we used `endAtFudged` in the filter instead, the results _wouldn't_ include flights starting on the 30th.